help/pt/Help_fdl-appendix.xml

```diff
@@ -593,7 +590,7 @@
     <title>9. TERMINATION</title>
     <para>
       You may not copy, modify, sublicense, or distribute the <link
-      linkend="fdl-document">Document</link> except as expressly
+      linkend="fdl-document">Document</link> excepto as expressly
```

Comment:
The license is only valid in english. But you can link the original and add an unofficial translation. Search the fsf.org for accepted translations.